packages/compiler/test/config/config-interpolation.test.ts

```diff
@@ -139,46 +168,31 @@ describe("compiler: config interpolation", () => {
       });
     });
 
-    it("expand nested variables", () => {
+    it("use parameter with {output-dir} in emitter-output-dir", () => {
       const config = {
         ...defaultConfig,
         projectRoot: "/dev/ws",
-        outputDir: "{test-var.one.x}/my-output",
+        outputDir: "{project-root}/my-output",
         parameters: {
           "test-var": {
-            one: {
-              x: "nested/test",
-            },
-            default: "",
+            default: "{output-dir}/test-var",
           },
         },
-      };
-      const resolved = expectExpandConfigVariables(config, { cwd: "/dev/wd" });
-      deepStrictEqual(resolved, {
-        ...config,
-        outputDir: "nested/test/my-output",
-      });
-    });
-
-    it("expand nested variables with default value", () => {
```

Comment:
those tests where testing an invalid config(they should have been removed when we found out about this misuse)